extensions/cms/src/cmsResource/cmsResourceCommands.ts

```diff
@@ -89,11 +89,18 @@ export function deleteRegisteredServerCommand(appContext: AppContext, tree: CmsR
 		if (!(node instanceof RegisteredServerTreeNode)) {
 			return;
 		}
-		appContext.apiWrapper.removeRegisteredServer(node.name, node.relativePath, node.ownerUri).then((result) => {
-			if (result) {
-				tree.notifyNodeChanged(undefined);
-			}
-		});
+		appContext.apiWrapper.showWarningMessage(
+			`${localize('cms.confirmDelete', 'Are you sure you want to delete ')}${node.name}?`,
```

Comment:
>  [](start = 68, length = 1)

Suggest leaving the space outside the localized bit - it'd be easier to mistakenly leave the space out when localizing these strings